docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+# Scale and limits
+
+This document describes current scale and limits known in OSM as of 02/04/2021.
+
+## Considerations
+The scale limits documented here have to be put in light of current architecture. 
+Current architecture relies on a global broadcast mechanism that does not account for proxy configuration deltas, therefore all proxy configurations are computed and pushed upon any change.
+
+## Testing and measures
+We currently hold a single test which attempts to scale infinitely a topology subset, test proper traffic configuration between the new pods/services being deployed in the iteration, and stop if any failure is seen.
+It is also acknowledged that some of the scale constraints need to be addressed before it even makes sense to proceed with any additional scale testing, hence the lack of additional test scenarios.
+
+## Test
+The test was run in different OSM form factors, factoring in different amounts of RAM/CPU, to better qualify potential limits in case any of those were to be a constrain upon deployment.
+
+#### For all tests:
+- Change ID 9b829667f797879a0a4cd911eb08c4808e0f8083 (Thu Feb 4 13:38:37 2021)
+- 8 Node (1.18, nodes: 4vcpu 16Gb)
+- Envoy proxy log level Error
+- OSM controller log level Trace (reasons)
+- HTTP debug server disabled on OSM
+- Test topology deploys each iteration:
+	- 2 clients
+		- 5 replicaset per client
+	- 5 server services
+		- 2 replicaset per server service
+	- 1 TrafficSplitt between the 5 server services (10 pods backed)
+	- Total of 20 pods each iteration. 10 client pods will REST GET the Traffic split.
+	- Correctness is ensured. It is checked that all TrafficSplitt server members are eventually reached.
+- Test timeout for network correctness: 150 seconds
+
+#### Varying form factors
+- with 256MB max RAM
+	- 1.5 CPU
+- With 512MB RAM
+	- 1.5 CPU
+	- 4 CPU
+
+(All tests were run multiple times to ensure validity of the results)
+
+## Assessment
+
+Note: Please assume we are deploying a proxy per pod in the following results, so use pod/proxies interchangeably.
+### Memory
+- Memory per pod/envoy onboarded in the network is calculated after the initial snapshot with nothing onboarded on the mesh is seen to take into account standalone memory used by OSM.
+  - Initial memory use of OSM **40~46MB**
+  - Memory increase per pod ranges between **800KB~1MB** per pod. 
+- With **256MB** max memory limit for OSM's container, **180-200 pods** should be a conservative limit. Any more pods
+can (and will) have OSM `OOMKilled` by Kubernetes.
+- With **512MB** max memory limit for OSM's container, it can reach to **~400 pods**, although convergence times will spike through the roof up to **+120s** with a rather oversubscribed CPU profile, that could potentially lead to other issues.
```

Comment:
> convergence times will spike through the roof up to **+120s**

How does this impact the workload?

Reply:
this directly implies formal delays til the network is fully settled and all smi is properly applied and pods are properly reachable based on latest configuration.